apps/api/src/app/widgets/pipes/limit-pipe/limit-pipe.ts

```diff
@@ -0,0 +1,36 @@
+import { ArgumentMetadata, BadRequestException, Injectable, PipeTransform } from '@nestjs/common';
+
+@Injectable()
+export class LimitPipe implements PipeTransform {
+  private readonly minInt: number;
+  private readonly maxInt: number;
+  private readonly isOptional: boolean = false;
```

Comment:
please add empty line between variables and constructor, and constructor and transform function